npm/Dockerfile

```diff
@@ -20,6 +20,9 @@ FROM docker.io/ubuntu:focal
 COPY --from=builder /usr/local/bin/azure-npm \
 	/usr/bin/azure-npm
 
+COPY --from=builder /usr/local/src/npm/npm/scripts \
```

Comment:
is `npm/npm/` intentional?

Reply:
Yep. Since I didnt want to change the WORKDIR in the builder image ([here](https://github.com/Azure/azure-container-networking/pull/1262/files/143d39929ebbd7773bb3a2a1662bc1c8137ec331#diff-9109d573bc356b2f5cfb6ccfa3a9a94643911cb8ef8e6b9de1e3a40debcf26f3R8-R11)) which copies the host's source to `/usr/local/src/npm` it has to be this way.